tests/installation/ipxe_install.pm

```diff
@@ -89,12 +89,12 @@ sub set_bootscript {
     }
 
     my $cmdline_extra;
-    $cmdline_extra .= " regurl=$regurl " if $regurl;
+    $cmdline_extra .= " regurl=$regurl " if ($regurl and !is_usb_boot);
     $cmdline_extra .= " console=$console " if $console;
 
     # Support passing both EXTRA_PXE_CMDLINE to bootscripts
     $cmdline_extra .= get_var('EXTRA_PXE_CMDLINE') . ' ' if get_var('EXTRA_PXE_CMDLINE');
-    $cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if check_var('IPXE_UEFI', '1');
+    $cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if (check_var('IPXE_UEFI', '1') and !is_usb_boot);
```

Comment:
I do not think you should make this change, because you never really install with these parameters typed in in ipxe_install.pm.

Reply:
I guess you misunderstand here?

Reply:
These two lines:

```
$cmdline_extra .= " regurl=$regurl " if ($regurl and !is_usb_boot);
$cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if (check_var('IPXE_UEFI', '1') and !is_usb_boot);
```
affect SelfInstall ISO host installation ? How ? I think you just boot SelfInstall ISO directly without modify any  boot parameters.

Reply:
See reply above.

Reply:
Same here. I do not think this command affects usb boot, because you do not need it at all, you just need ssh server up.

Reply:
Just as you said, just ssh up is enough, so exclude these settings to avoid potential impacts. The simpler, the less involved, the more stable it can  be.

Reply:
I totally appreciate the review and suggestions. And I fully understand that when reviewers give comment, they tend to let the author agree and make corresponding change. Meanwhile, as the  PR author, I have well read the comment and think about that and shared the reasons/sense for keeping current way. Since it is trivial comment, it won't make much difference, and everyone's effort is limited, I'd suggest to respect the author's choice a little bit more, and stop further discussion on it.
I know there are several other minor comments which have similar situation. I won't reply one by one with above words, but similar decision is made with similar reasons, just as the "Resolved conversation " tells. Hope you understand with more tolerance ;-). We have to admit that people have different styles. Let's get major agreements with minor differences kept. This also shows good mutual respect. Take this also as a minor suggestion that I give for future review to any other's PRs :). Thanks again!